flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -171,6 +171,11 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
 
     val appStartMetrics = AppStartMetrics.getInstance()
 
+    if (!appStartMetrics.isAppLaunchedInForeground) {
```

Comment:
this is good
we also have [another guard](https://github.com/getsentry/sentry-java/blob/61c8d80abb339b6ed87466e712d5a8d06aa97fba/sentry-android-core/src/main/java/io/sentry/android/core/performance/AppStartMetrics.java#L278-L286) in Android (which should be useful for iOS, too): if the appStartSpan takes more than 1 minute, we drop it
Can you add it?

Reply:
@stefanosiano thanks for the hint. 
I changed it to 
```kotlin
if (!appStartMetrics.isAppLaunchedInForeground ||
  appStartMetrics.appStartTimeSpan.durationMs > 1.toDuration(DurationUnit.MINUTES).inWholeMilliseconds
) {
  Log.w(
    "Sentry",
    "Invalid app start data: app not launched in foreground or app start took too long (>60s)",
  )
  result.success(null)
}
```
Did you mean it that way?

Reply:
yep